ingest-mongodb-public/environments/production.yml

```diff
@@ -5,7 +5,14 @@ image:
 cronJobs:
   - name: ingest-all-content
     schedule: "0 6 * * *" # every day at 6am UTC (1AM EST)
-    command: ["node", ".", "all"]
+    command:
+      [
+        "node",
+        "./node_modules/ingest/build/main.js",
+        "all",
+        "--config",
+        "./build/config.js",
+      ]
```

Comment:
maybe instead we can make a package.json script for this. smthn like: 

```shell
npm run ingest:all
```

and that script would look smthn like:

```json
"ingest:all": "mongodb-rag-ingest-cli all --config ./build/config.js"
```